### ISSUE TITLE
I don't believe i am getting all necessary settings for JappoxMini

### ISSUE THREAD

**Issue by bytedeez** [friendica/friendica#1611]
![jappix settings](https://cloud.githubusercontent.com/assets/7134028/7802982/3e7dc77c-0311-11e5-91b9-46d09893a3fe.png)

Also can someone tell me the cron job it is speaking off? i have already set the poller.php cron

**Comment by rabuzarus** [comment]
Your settings will not work. The bosh proxy from jappix (bind.jappix.com) work only with their own accounts. If you want to use the jappix bosh proxy you have to sign up a jabber account on jappix.com. Otherwise you have to chose another bosh proxy or setup your own

**Comment by bytedeez** [comment]
Thanks! is there a way to trouble shoot errors on jappix other than it the error chat box sending me to mini.jappix.com?

**Comment by rabuzarus** [comment]
At least I don't know anyone

**Comment by bytedeez** [comment]
In my account settings page i can use a separate Jabber Server and bosh host correct?

**Comment by rabuzarus** [comment]
Yes this correct, but you have to find a bosh proxy which accept your jabber server or you have to setup your own bosh proxy

**Comment by fabrixxm** [comment]
@bytedeez  What's the status of this?

**Comment by bytedeez** [comment]
Although from reading the wiki I have my settings correct. I am still erroring out.
